config/initializers/sandbox_email_interceptor.rb

```diff
@@ -0,0 +1,8 @@
+if Rails.env.development?
+  class SandboxEmailInterceptor
+    def self.delivering_email(message)
+      message.to = ['hyosunko@gmail.com']
```

Comment:
It might be worth storing this backup email (and the other default admin and notifications Coding Zeal emails) in the environment variables file. That way, you keep your real contact info out of the app in a secret secure place--just make sure you share these new env values with your teammates and add them on Heroku. This is just a suggestion though.

Reply:
oh..it was used to intercept email beginning of development, and later we changed to mail catcher. So it was not needed any more. I will delete this file. I will discuss with Adam if he want to have backup emails.